apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -604,4 +617,18 @@ const DanceAiModal: React.FunctionComponent<DanceAiProps> = ({onClose}) => {
   );
 };
 
+const getTranslationsFromField = (
```

Comment:
I could see this being useful outside of Dance AI as well, as a way to pull dropdown labels from any dropdown field - thoughts on pulling this into a separate file?

Reply:
Was just looking to see if there's a blockly utils kind of place to put this, although I don't wanna lose my nice typescript types!

Reply:
Yep, not a strong preference, maybe we can just move to `dance/ai/utils`?

Reply:
Done! https://github.com/code-dot-org/code-dot-org/pull/54534/commits/4650bb89354e9b7e18bdc3df0edced5977943e13